src/potentials.h

```diff
@@ -8,27 +8,138 @@
 #include "functionparser.h"
 #include "multipole.h"
 #include <array>
+#include <functional>
+#include "spdlog/spdlog.h"
+#include "spdlog/sinks/null_sink.h"
 
 namespace Faunus {
 namespace Potential {
 
 using namespace std::string_literals;
 
+//! type of a matrix containing pair potential coefficients
+typedef Eigen::MatrixXd TPairMatrix;
+typedef std::shared_ptr<TPairMatrix> TPairMatrixPtr;
```

Comment:
these typedefs seem to be connected with the mixer yet appear in the enclosing namespace. Why not confine them in `PairMixer`?

Reply:
Both the `PairMixer` class and descendants of `MixerPairPotentialBase` class share the equal interest in this data type. I would argue that this data type is meaningful even without the existence of the `PairMixer` class as the matrices can be produced in other ways.